.github/workflows/test.yml

```diff
@@ -0,0 +1,72 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '5 15 * * SUN'
+
+jobs:
+  build-job:
+    name: Build distribution
+    runs-on: ubuntu-latest
```

Comment:
If you go with `ubuntu-20.04` then you won't get a bunch of annoying annotations on your build about the fact that `ubuntu-latest` is set to change.